src/eei.cpp

```diff
@@ -61,6 +61,25 @@ Literal EthereumInterface::callImport(Import *import, LiteralList& arguments) {
       return Literal();
     }
 
+    if (import->base == Name("getBalance")) {
+      std::cout << "getbalance\n";
+
+      uint32_t addressOffset = arguments[0].geti32();
+      uint32_t resultOffset = arguments[1].geti32();
+
+      evm_address *address = new evm_address;
+      evm_uint256be *result = new evm_uint256be;
```

Comment:
If `result` is passed to C as a reference this doesn't need to be allocated either.